openmetadata-ui/src/main/resources/ui/src/pages/DashboardDetailsPage/DashboardDetailsPage.component.tsx

```diff
@@ -746,6 +746,27 @@ const DashboardDetailsPage = () => {
     updateThreadData(threadId, postId, isThread, data, setEntityThread);
   };
 
+  const handleExtentionUpdate = async (updatedDashboard: Dashboard) => {
+    try {
+      const response = await saveUpdatedDashboardData(updatedDashboard);
+      const data = await response.data;
```

Comment:
```suggestion
    try {
      const {data} = await saveUpdatedDashboardData(updatedDashboard);
```

Reply:
@ShaileshParmar11, Here we are using the async/await pattern and we have to await for the data from the response object, so we have to do that way.

Reply:
ya but this will work, let me know if you need help!

Reply:
Yeah, it will work but we should have await for data there.

Reply:
@ShaileshParmar11 , Got your point , I will make the changes :)